server/src/game/components/velocity.rs

```diff
@@ -0,0 +1,8 @@
+
+///Stores inforamtion about an entities velocity in meters/seconds.
+pub struct Velocity{
+    ///the x-axis velocity in meters/seconds.
+    pub dx : f32,
```

Comment:
should prob be decimal and maybe velocity should be commen for both x and y

Reply:
also should we have an acceleration?

Reply:
Decimal? You meen f64 instead of f32?

Reply:
Acceleration would be fun to do, that would make it harder to "change direction" on long flights.